metrics/deploy/lib/doctor/doctor_command.dart

```diff
@@ -1,21 +1,36 @@
-import 'dart:io';
-
 import 'package:args/command_runner.dart';
-import 'package:process_run/process_run.dart' as cmd;
+import 'package:deploy/cli/firebase/firebase_command.dart';
+import 'package:deploy/cli/flutter/flutter_command.dart';
+import 'package:deploy/cli/gcloud/gcloud_command.dart';
+import 'package:deploy/cli/git/git_command.dart';
 
-/// class providing doctor command to verify dependencies.
+/// A class providing doctor command to verify dependencies.
 class DoctorCommand extends Command {
   @override
   final name = "doctor";
   @override
   final description = "Check dependencies.";
 
+  /// Provides Firebase CLI.
+  final _firebase = FirebaseCommand();
+
+  /// Provides GCloud CLI.
+  final _gcloud = GCloudCommand();
+
+  /// Provides Git CLI.
+  final _git = GitCommand();
+
+  /// Provides Flutter CLI.
+  final _flutter = FlutterCommand();
+
+  /// Creates this doctor command instance.
```

Comment:
Maybe we can remove this constructor since the class has an empty constructor by default?